lib/index-static.jsx

```diff
@@ -15,6 +15,8 @@ function generateWithPageHTML(url, options, pageHTML) {
         <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,400italic,600,700,600italic,700italic,800,800italic"/>
         <link rel="stylesheet" href="/vendor/bootstrap/css/bootstrap.min.css"/>
         <link rel="stylesheet" href={'/' + exports.CSS_FILENAME}/>
+        <script src='https://api.tiles.mapbox.com/mapbox.js/v2.1.5/mapbox.js'></script>
```

Comment:
Hmm, at the very least we should move the script tag to the end of the document for faster page loading... After we land this, though, we might want to file a ticket later to only load this on the Clubs page.